src/LogConfigFile.cpp

```diff
@@ -13,14 +14,19 @@ std::ostream& operator<<(std::ostream& ostream, const Location& location)
 {
 	ostream << "Path: " << location.path << '\n';
 	ostream << "Root: " << location.root << '\n';
-	ostream << "Index: " << location.index << '\n';
+	ostream << "Index:\n";
+	for (std::vector<std::string>::const_iterator it = location.indices.begin(), end = location.indices.end(); it != end; ++it)
+		ostream << *it << '\n';	
 	ostream << "CGI extension: " << location.cgiExt << '\n';
 	ostream << "CGI path: " << location.cgiPath << '\n';
 	ostream << "Autoindex: " << location.isAutoindex << '\n';
-	ostream << "LimitExcept:\n";
-	ostream << "  Allow: " << location.limitExcept.allow << '\n';
-	ostream << "  Deny: " << location.limitExcept.deny << '\n';
-	ostream << "  Allowed methods:\n";
+	ostream << "Max body size: " << location.maxBodySize << '\n';
+	ostream << "Error pages:\n";
+	for (std::map<unsigned short, std::string>::const_iterator it = location.errorPage.begin(), end = location.errorPage.end(); it != end; ++it)
+		ostream << it->first << ": " << it->second << '\n';	
+	ostream << "Allowed methods:\n";
+	for (int i = 0; i < MethodCount - 1; ++i)
+		ostream << "  " << location.allowedMethods[i] << '\n';	
```

Comment:
clang-tidy: avoid array access. Use allowedMethods.at(i) instead.

Reply:
To print out the allowed methods including their current status (true or false) is now implemented in this way:

```cpp
ostream << "Allowed methods:\n";
ostream << "Get: " << location.allowedMethods[MethodGet] << '\n';
ostream << "Post: " << location.allowedMethods[MethodPost] << '\n';
ostream << "Delete: " << location.allowedMethods[MethodDelete] << '\n';
```